packages/react/src/VideoPlayer/VideoPlayer.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import {render, cleanup, fireEvent} from '@testing-library/react'
+import '@testing-library/jest-dom'
+
+import {VideoPlayer} from '.'
+
```

Comment:
I think more tests could be included here but I haven't been able to get specific state values to update. If anyone has some ideas for improvements please let me know.

Reply:
Which state are you looking to update?

Reply:
I would love to pair on it. I specifically can't get the seek state to update in a way that I am able to test. I am free tomorrow morning if you have some time I will make an invite.